Ship_Game/GameScreens/GameScreen.cs

```diff
@@ -189,6 +189,13 @@ protected int SelectSize(int lowRes, int normalRes, int hiRes)
         public void AddLight(ILight light, bool dynamic) => ScreenManager.AddLight(light, dynamic);
         public void RemoveLight(ILight light, bool dynamic) => ScreenManager.RemoveLight(light, dynamic);
 
+        protected virtual void ResetMusic()

```

Comment:
If you turn this into protected virtual, then it would imply that ResetMusic() is somehow automatically handled by GameScreen. However you made it virtual and then explicitly call it in Shipyard.

Since all of this is actually managing `ScreeManager.Music` AudioHandle, it would make more sense to move this logic into `ScreenManager`

Add a `StopMusic()` method into ScreenManager.
And for MainMenu/UniverseScreen/ShipdesignScreen there would be a unique StartMusic function instead.

Reply:
done